addons/l10n_in_pos/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_account_tag_user,account.account.tag.user,account.model_account_account_tag,base.group_user,1,0,0,0
```

Comment:
Why `base.group_user`?
If it is indeed correct, why would it not be in `account` directly?

Reply:
It's used in l10n_in_pos so here.
I will move it to `account` if you suggest